src/parser/MaintainSentences.h

```diff
@@ -76,11 +59,124 @@ class ColumnSpecificationList final {
 };
 
 
+class SchemaPropItem final {
+public:
+    using Value = boost::variant<int64_t, bool, std::string>;
+
+    enum PropType : uint8_t {
+        TTL_DURATION,
+        TTL_COL,
+    };
+
+    SchemaPropItem(PropType op, int64_t val) {
+        propType_ = op;
+        propValue_ = val;
+    }
+
+    SchemaPropItem(PropType op, bool val) {
+        propType_ = op;
+        propValue_ = val;
+    }
+
+    SchemaPropItem(PropType op, std::string val) {
+        propType_ = op;
+        propValue_ = std::move(val);
+    }
+
+    StatusOr<int64_t> getTtlDuration() {
+        if (isInt()) {
+            return asInt();
+        } else {
+            LOG(ERROR) << "Ttl_duration value illegal: " << propValue_;
+            return Status::Error("Ttl_duration value illegal");
+        }
+    }
+
+    StatusOr<std::string> getTtlCol() {
+        if (isString()) {
+            return asString();
+        } else {
+            LOG(ERROR) << "Ttl_col value illegal: " << propValue_;
+            return Status::Error("Ttl_col value illegal");
+        }
+    }
+
+    PropType getPropType() {
+        return propType_;
+    }
+
+    std::string toString() const;
+
+private:
+    int64_t asInt() {
+        return boost::get<int64_t>(propValue_);
+    }
+
+    const std::string& asString() {
+        return boost::get<std::string>(propValue_);
+    }
+
+    bool asBool() {
+        switch (propValue_.which()) {
+            case 0:
+                return asInt() != 0;
+            case 1:
+                return boost::get<bool>(propValue_);
+            case 2:
+                return asString().empty();
+            default:
+                DCHECK(false);
+        }
+        return false;
+    }
+
+    bool isInt() {
+        return propValue_.which() == 0;
+    }
+
+    bool isBool() {
```

Comment:
ditto

Reply:
Because the type of some values can be bool type.